README.md

```diff
@@ -22,6 +22,36 @@ Every ID (except systemId, which (by GBFS convention) should already be globally
 At least every operator should be assigned its own unique codespace, assuming operators already maintain data / ID separation accross their feeds.
 The codespaced operatorId should comply with the following format `<codespace>:Operator:<operator>`.
 
+#### Configuring Authentication
+
+Some providers may require authentication, e.g. via bearer token, OAuth2 or custom http-headers.
+
+    lamassu:
+        providers:
+          - systemId: my_oauth2_secured_system
+            authentication:
+                scheme: oauth2ClientCredentialsGrant
```

Comment:
Does this parse into the enum correctly? I use the SCREAMING_SNAKE_CASE variant in my config - I think it's better to align with the enum values.

Reply:
Changed them to SCREAMING_SNAKE_CASE.